### PR TITLE
fix: make the workspace-selector label smaller

### PR DESCRIPTION
**Pull request by gabyzif**
## Summary

This PR makes the workspace selector label to match the size of the rest of the nav labels.

## Testing Plan

- [ ] Was this tested locally? If not, explain why.
- {explain how this has been tested, and what, if any, additional testing should be done}

## Reference Issue (For mParticle employees only. Ignore if you are an outside contributor)

- Closes https://mparticle-eng.atlassian.net/browse/UNI-831

### CODE REVIEW

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 1]
```diff
@@ -251,7 +252,7 @@
 
 .globalNavigation__iconLabel {
   color: var(--color-text);
-  font-size: 10px;
+  font-size: var(--label-size);
```
Comment (tibuurcio) [inline src/components/navigation/GlobalNavigation/global-navigation.css:255]: How is it smaller if the variable has the same value as before? (10px). I'm confused 😅

Comment (nastyastavitskaya) [reply src/components/navigation/GlobalNavigation/global-navigation.css:255]: this is the correct value, Gaby just replaced it with a variable. the workspace selector text was not correct and was suppose to match this font-size

**Review by tibuurcio — APPROVED** [review]

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 2]
```diff
@@ -94,7 +95,7 @@
 }
 
 .globalNavigation__item.workspaceSelector__menuItem{
-  height: var(--nav-item-height-large) !important;
+  height: calc(var(--control-height-lg) * 2) !important;
```
Comment (nastyastavitskaya) [inline src/components/navigation/GlobalNavigation/global-navigation.css:98]: can we remove `var(--nav-item-height-large)` if it's no longer used?

**Review by nastyastavitskaya — APPROVED** [review]
looks so much better! 👏 🎉

**Pull request merged**